lib/l10n/app_en.arb

```diff
@@ -635,5 +635,7 @@
   "devConfirmProposal": "Confirm Proposal",
   "devProposalComplete": "PROPOSAL COMPLETE",
   "devEnableMultiSig": "Enable Multi Sig",
-  "sendAssetListLoadingAssets": "Loading assets"
+  "sendAssetListLoadingAssets": "Loading assets",
+  "pleaseNote": "Please note",
```

Comment:
It's been the standard to prefix the name of the screen to the name of the String, like so:
`"pleaseNote": "Please note",` -> `"multiSigRecoverPleaseNote": "Please note",`

Reply:
ok will make that change